File: python/fizzbuzz/fizzbuzz.py
def fizzbuzz(number=100):
    fizzbuzz = '';
    for i in range(1, number + 1):
        if i % 3 == 0:
            fizzbuzz += 'Fizz'
        if i % 5 == 0:
            fizzbuzz += 'Buzz'
        if i % 3 !=0 and i % 5 != 0:
            fizzbuzz += str(i)

        fizzbuzz += '\n'

    return fizzbuzz

File: haskell/fizzbuzz.hs
main :: IO()
main = mapM_ putStrLn $ fizzDemThings 100

fizzDemThings :: Int -> [String]
fizzDemThings numbah = take numbah (map fizzThatThang [1..])

fizzThatThang :: Integer -> String
fizzThatThang numbah | doesItFizz numbah = "Fizz"
                     | doesItBuzz numbah = "Buzz"
                     | doesItBlow numbah = "FizzBuzz"
                     | otherwise         = show numbah

doesItFizz :: Integer -> Bool
doesItFizz numbah = (&&) (0 == numbah `mod` 3) (not (doesItBlow numbah))

doesItBuzz :: Integer -> Bool
doesItBuzz numbah = (&&) (0 == numbah `mod` 5) (not (doesItBlow numbah))

doesItBlow :: Integer -> Bool
doesItBlow numbah = 0 == numbah `mod ` 15
